packages/reporter/src/commands/command.tsx

```diff
@@ -77,7 +93,7 @@ interface AliasesReferencesProps {
 const AliasesReferences = observer(({ model, aliasesWithDuplicates }: AliasesReferencesProps) => (
   <span>
     {_.map(([] as Array<AliasObject>).concat((model.referencesAlias as AliasObject)), (aliasObj) => (
-      <span className="command-alias-container" key={aliasObj.name + aliasObj.cardinal}>
+      <span className='command-alias-container' key={aliasObj.name + aliasObj.cardinal}>
```

Comment:
shouldn't this remain double quotes?, not sure if the linter cares, but i though we generally use double quotes in jsx

Reply:
The linter doesn't care, though I agree, I am partial to the double quotes, but I think that's engrained from the Terra linter rules we had established previously. If you look through the various JSX components in the reporter, single quotes are consistently used.